pkgs/top-level/all-packages.nix

```
@@ -21994,7 +21994,8 @@ in
     pythonPackages = python3Packages;
   };
 
-  way-cooler = callPackage ../applications/window-managers/way-cooler {};
+  way-cooler = throw ("Project abandoned by its author: " +
```

```suggestion
  way-cooler = throw ("way-cooler is abandoned by its author: " +
```

else this message just confuses people